example/lib/main.dart

```diff
@@ -34,9 +35,9 @@ class _MyAppState extends State<MyApp> {
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             AdvancedCalendar(
-              controller: _calendarControllerToday,
-              events: events,
-            ),
+                controller: _calendarControllerToday,
+                events: events,
+                startWeekDay: startWeekDay),
```

Comment:
It's not useful to set a date as a start week day, it should be a number [0-6] or enum {sun, mon, tue, ...}